### PR TITLE
feat: added timers for defrag powerups

### PR DESCRIPTION
**Pull request by Mac-tf2**
Closes momentum-mod/game/issues/1848
Closes momentum-mod/game/issues/1727

### Checks

-   [x] **I have thoroughly tested all of the code I have modified/added/removed to ensure something else did not break**
-   [x] I have followed [semantic commit messages](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716) e.g. `feat: Add foo`, `chore: Update bar`, etc...
-   [x] My branch has a clear history of changes that can be easy to follow when being reviewed commit-by-commit
-   [x] My branch is functionally complete; the only changes to be done will be those potentially requested in code review
-   [ ] All changes requested in review have been `fixup`ed into my original commits.

### CODE REVIEW

**Comment by Mac-tf2** [comment]
image

File: layout/hud/powerup-timer.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		<include src="file://{resources}/styles/main.scss" />
+	</styles>
+	<scripts>
+		<include src="file://{scripts}/common/gamemodes.js" />
+		<include src="file://{scripts}/util/register-event-for-gamemodes.js" />
+		<include src="file://{scripts}/hud/powerup-timer.js" />
+	</scripts>
+
+	<MomHudPowerupTimer class="powerup-timer">
+		<Panel id="HasteTimer" class="powerup-timer__item">
```
Comment (tsa96) [inline layout/hud/powerup-timer.xml:12]: Set visible=false to all of thse

**Review by tsa96 — APPROVED** [review]

File: layout/hud/powerup-timer.xml (hunk 1 above)
Comment (tsa96) [reply layout/hud/powerup-timer.xml:12]: Nvm

**Comment by tsa96** [comment]
I'm happy with Mac's stuff but seeing as I rewrote half of it with him and this PR now has a commit of my own, def needs a review from someone else - Peen if you can check it out that'd be great. Also I have a PR on red for getting the ShouldDraw stuff for hud panels working with my thing.

**Pull request merged**